src/parse.ts

```diff
@@ -224,10 +224,25 @@ function typeContentToTsHelper(
         ) {
           // If there is a description it means we also have a new line, which means
           // we need to properly indent the following line too.
-          const prefix = descriptionStr != '' ? descriptionStr : first ? '' : skipNewline ? '' : '\n';
+          let prefix = descriptionStr;
+          if (prefix == '') {
```

Comment:
Can you make this ===
There are a few places that need this update

Reply:
Applied the ESLint rule!